R/pruneRepo.R

```diff
@@ -56,36 +74,49 @@ pruneRepo <- function(repopath = getOption("dratRepo", "~/git/drat"),
     } else {
         stop("Unknown package type", call. = FALSE)
     }
+    ext
+}
+
+.get_repoinfo_per_package_type <- function(t, e, rd, pkg){
+    files <- list.files(rd, pattern = e, full.names = FALSE)
+    if(length(files) == 0L){
+        return(data.frame())
```

Comment:
maybe better 0 rows data.frame but still providing empty columns? then less exception to handle when processing output of this function...

Reply:
It is filtered out by using 'nrow() == 0L'. See line 58. Could also be done by returning `NULL` in this case, but other cases of zero-row data.frames exist in that function as well. This solution takes care of all of them.

Reply:
ok, it is internal is should be fine